qt/python/CMakeLists.txt

```diff
@@ -5,14 +5,34 @@
 # Legacy wrappers for MantidPlot
 add_subdirectory ( mantidqtpython )
 
-# mantidqt is run from the source directory so just add tests
-set ( PYTHON_TEST_FILES
-  mantidqt/test/import_test.py
-)
+###########################################################################
+# mantidqt
+###########################################################################
 
-# Tests
-pyunittest_add_test ( ${CMAKE_CURRENT_SOURCE_DIR}
-  mantidqt ${PYTHON_TEST_FILES}
-)
+if ( ENABLE_WORKBENCH )
 
-# No package installation yet...
+  # Configure utils.qt.plugins file for build. It is placed in the source
+  # directory and added to the .gitignore for simplicity.
+  if ( WIN32 )
+  set ( QT_PLUGINS_PATH "${THIRD_PARTY_DIR}/lib/qt%V/plugins" )
+  endif ()
+  configure_file ( mantidqt/utils/qt/plugins.py.in
+    ${CMAKE_CURRENT_SOURCE_DIR}/mantidqt/utils/qt/plugins.py
+  )
+
+  # Create egg link to binary output directory for mantidqt
+  add_python_package ( mantidqt )
+
+  # ctest targets
+  set ( PYTHON_TEST_FILES
+    mantidqt/test/import_test.py
```

Comment:
Did you mean to move this test to only run when the workbench is enabled?

Should we set `ENABLE_WORKBENCH=ON` on pull requests? We just need to install the required dependencies.

Reply:
I had originally intended to only have the `mantidqt` module in use for the workbench but following the TSC discussion there is no reason it cannot be available for both versions.

I would like to get `ENABLE_WORKBENCH=ON` for the pull requests too. Are we just waiting on PyQt5 for RHEL?

Reply:
👍 We can enable it later once we get all the dependencies.